docs/requirements/firewall.md

```diff
@@ -14,8 +14,9 @@ Dedicated devices are often behind corporate firewalls, preventing Esper's backe
 | mqtt-telemetry-prod.esper.cloud       | TCP: 1883 (MQTT)                     | Deep telemetry from devices                                                              |
 | IP: 13.52.132.230          |  TCP: 40000 - 50000 | For allowing secure remote ADB access to your devices                       |
 | firebaseinstallations.googleapis.com and fcm.googleapis.com| TCP: 443 (HTTPS), TCP: 5228 (HTTPS), TCP: 5229 (HTTPS), TCP: 5230 (HTTPS)  | Backup channel, also used for ping command to wake up a device (Only for GMS devices)       |
+|IP: 8.8.8.8                           |	TCP: 443 (HTTPS)                   |	For verifying internet connectivity on the device                             |
 
-
-:::tip
-For static IP, please get in touch with the [Esper team](https://support.esper.io/s/).
-:::
\ No newline at end of file
+:::
+tip
```

Comment:
@xiliao31  tip should be in the same line like 
![image](https://user-images.githubusercontent.com/12209601/192927319-bf4edb2a-3cde-429a-99d1-2632181caa01.png)

Please refer to this for detailed documentation
https://v1.vuepress.vuejs.org/guide/markdown.html#table-of-contents